sql/core/src/test/scala/org/apache/spark/sql/execution/command/PlanResolutionSuite.scala

```diff
@@ -808,11 +808,11 @@ class PlanResolutionSuite extends AnalysisTest {
 
     // For non-existing tables, we convert it to v2 command with `UnresolvedV2Table`
     parsed4 match {
-      case AlterTableSetProperties(_: UnresolvedTable, _) => // OK
+      case SetTableProperties(_: UnresolvedTable, _) => // OK
       case _ => fail("Expect AlterTableSetProperties, but got:\n" + parsed4.treeString)
```

Comment:
Update this message?

Reply:
Same for other places.

Reply:
@imback82 Thank you. Maybe, we should replace it by `${SetTableProperties.getClass.getName}` to easily find it in IDE.